form.php

```diff
@@ -384,9 +384,11 @@ public function onFormProcessed(Event $event)
         $action = $event['action'];
         $params = $event['params'];
 
-        $this->process($form);
-
         switch ($action) {
+            case 'process':
+                $this->process($form);
```

Comment:
It appears with your change, the form will only be processed **IF** it has a:

```
process:
  process: true
  ...
```

This would mean that all existing forms would no longer automatically process the fields.. Granted we currently only support the `fillWithCurrentDateTime` task, but still, it would not work until the form blueprint was updated.